composer.json

```diff
@@ -10,13 +10,13 @@
 		"Basic Auth",
 		"Client Certificate Auth"
 	],
-	"homepage": "http://chroma-x.de/",
+	"homepage": "https://chroma-x.de/",
 	"license": "MIT",
 	"authors": [
```

Comment:
You should mention yourself there

Reply:
@Bonscho The update is complete. Please approve the change and release a new major version.